### DISCUSSION TITLE
I can not install bthps3 on win 10

### DISCUSSION THREAD

**Discussion by ygv627** [nefarius/BthPS3#98] [Support]
![image](https://github.com/nefarius/BthPS3/assets/159639126/a3aedd99-3d4c-409a-9ce1-413bd41ce331)

**Comment by ygv627** [comment]
![image](https://github.com/nefarius/BthPS3/assets/159639126/36dfe65b-5b5a-486f-a926-5a77911004a2)

**Reply by nefarius** [reply]
Please run this line in Terminal/PowerShell and report back the result:

```PowerShell
Get-ItemPropertyValue -Path 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion' -Name CurrentMajorVersionNumber
```

Cheers

**Comment by ygv627** [comment]
![image](https://github.com/nefarius/BthPS3/assets/159639126/01a948d9-2967-4a20-9434-0416780fcf31)

**Comment by ygv627** [comment]
![image](https://github.com/nefarius/BthPS3/assets/159639126/98409911-7874-4978-9b71-cc9a4f0fdfe3)
